src/main/java/com/dmgrt/cinema/util/HibernateUtil.java

```diff
@@ -0,0 +1,20 @@
+package com.dmgrt.cinema.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
```

Comment:
see checklist: add a private default constructor to prevent HibernateUtil’s objects creating